src/components/profile/profileInfo/profileInfo.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Avatar } from '@material-ui/core';
+
+const ProfileInfo = () => {
+  const avatar = useSelector((state) => state.profile.avatar);
+  const fullname = useSelector((state) => state.profile.fullname);
+  const occupation = useSelector((state) => state.profile.occupation);
+  const email = useSelector((state) => state.profile.email);
+
+  return <div>
+    {avatar ? fullname : <Avatar></Avatar>}
+    <h2>{fullname ? fullname : 'Fullname'}</h2>
```

Comment:
better to show nothing if fullname is empty

Reply:
I've wrote this like a jammer